content/en/user-guide/aws/kinesis/index.md

```diff
@@ -43,11 +43,11 @@ LocalStack supports two third-party providers for Kinesis:
 * [kinesis-mock](https://github.com/etspaceman/kinesis-mock)
 * [kinesalite](https://github.com/mhart/kinesalite)
 
-By default `kinesis-mock` is used.
-Your desired provider can be set with the environment variable `KINESIS_PROVIDER`.
-For instance, if you wish to use `kinesalite` you'd pass the environment variable as `KINESIS_PROVIDER=kinesalite`.
-While both providers are supported, we recommend working with `kinesis-mock`, as it is more actively maintained.
-Moreover, the [Cloud Pods]({{< ref "cloud-pods" >}}) feature provides support **only** for `kinesis-mock`. 
+By default LocalStack uses `kinesis-mock`. While both providers are supported, we recommend working with `kinesis-mock`, as it is more actively maintained. Moreover, the [Cloud Pods]({{< ref "cloud-pods" >}}) feature provides support **only** for `kinesis-mock`. 
```

Comment:
The support for `kinesalite` was deprecated already and is completely dropped with 1.3, so it's not possible anymore at all to use `kinesalite` in localstack.